### ISSUE TITLE
encoding error

### ISSUE THREAD

**Issue by mokkin** [mautic/mautic#13404]
### Mautic Version

5.0.x series

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.1.27

### What browsers are you seeing the problem on?

Firefox

### What happened?

Encoding of `&` not working in settings page:

![grafik](https://github.com/mautic/mautic/assets/2938748/b5cf704e-0aea-4a6c-af31-341f986b591e)

### How can we reproduce this issue?

Just visit settings page at https://mautic.example.de/s/config/edit

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by mokkin** [comment]
The same applies to umlauts such as `ö`:
![grafik](https://github.com/mautic/mautic/assets/2938748/3d3d9352-c951-43b3-995d-dbb0cb32545d)

**Comment by RCheesley** [comment]
Hi there,

According to the reports on this issue: https://github.com/mautic/mautic/issues/13639 (duplicated by this one), the issue was resolved in 5.1. Therefore I'll close, but let us know if that's not the case please by raising another issue!
